apps/k8s/kubepack/manifests/output/install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+
+pushd manifests/output/github.com/kubepack/aws-microservices-deploy-options/apps/k8s/kubepack
```

Comment:
Should this be referring to `kubepack` account? Can make this script account independent?